content/user/ui/settings/Groups.md

```diff
@@ -1,74 +1,65 @@
+Contrast provides role based access control (RBAC) capabilities through groups. RBAC is an approach used to assign system, organization and application access to authorized users. Contrast has two types of groups: Organization Groups, which are used to assign organization and application access, and System Groups, which are used to assign system access. System Groups are only available to on-premise customers.
+
+## Organization Groups
+
+Contrast uses Organization Groups to assign authorized users access to organizations and applications. You can create Organization Groups in the System Admin interface as well as within a specific organization in the Organization Settings interface.
+
+### Overview of the Organization Groups interface
+
+The Groups interface is accessed from Organization Settings. As an Organization Administrator, you can create, edit and delete groups from this interface. In the overview, you can see all the groups that affect only the organization you are logged into, the application role provided by the group, as well as how many users and applications are managed. Hovering over the details in each group provides information about the applications, users and the roles provided to the group members.
+
+Each organization has four default groups: View, Edit, Rules Admin and Admin. These groups provide access to all applications in the organization with the associated role. In other words, if a user is placed in the Admin group, that user has the application administrator role for all applications in the organization. Users should only be placed in one of the four default groups at a time. Users can be added or removed from these default groups, but they can’t have their applications or roles edited and they can’t be deleted.
+
+### Creating an Organization Group from Organization Settings
+
+Groups created from the Organization Settings interface only impact the roles and permissions of the organization in which they are created. 
+
+> For example, let’s say that you have two organizations in Contrast: OrgA and OrgB. When you log into OrgB and create a group in OrgB, the permissions and roles that you define affect only OrgB users and OrgB applications.
+
+To create a group, you must have an Admin role in the organization in which you want to create the group. Once you log in as an Admin, navigate to the Organization Settings interface and then to the Groups interface.
+
+Begin creating your group by defining the level of access that you want the group to provide. Groups define access as a pairing of application(s) and role. In the applications field, select a single application, multiple applications or all applications. Then select the role you want to provide to those applications. The available roles are Admin, Rules Admin, Edit, View and No Access. 
+
+For more information, read the article on [Roles](https://docs.contrastsecurity.com/user_tsguideset.html#roles).
```

Comment:
Link format doesn't need to include the full path - see the format in the archived version - should just be (user_tsguideset.html#roles)